### PR TITLE
Assignment2 RDF RDFS TTL

### PR DESCRIPTION
**Pull request by CarlTloz**

### CODE REVIEW

File: Assignment2/CarlTloz-CarlosSolar.ttl [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@prefix rdf <http://www.w3.org/1999/02/22-rdf-syntax-ns>
+:User10A hasName :"Pedro".
+:Computer101 hasOwner :User10A.
+:Measurement8401 hasTemperature :"29";
+				 atTime :"2010-06-12T12:00:12".
```
Comment (ocorcho) [inline Assignment2/CarlTloz-CarlosSolar.ttl]: Literal should be datatyped

File: Assignment2/CarlTloz-CarlosSolar.ttl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@prefix rdf <http://www.w3.org/1999/02/22-rdf-syntax-ns>
+:User10A hasName :"Pedro".
+:Computer101 hasOwner :User10A.
+:Measurement8401 hasTemperature :"29";
```
Comment (ocorcho) [inline Assignment2/CarlTloz-CarlosSolar.ttl]: Literal should be datatyped

File: Assignment2/CarlTloz-CarlosSolar.ttl [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@prefix rdf <http://www.w3.org/1999/02/22-rdf-syntax-ns>
+:User10A hasName :"Pedro".
```
Comment (ocorcho) [inline Assignment2/CarlTloz-CarlosSolar.ttl]: Wrong literal

File: Assignment2/CarlTloz-CarlosSolar.ttl [hunk 4]
```diff
@@ -0,0 +1,8 @@
+@prefix rdf <http://www.w3.org/1999/02/22-rdf-syntax-ns>
```
Comment (ocorcho) [inline Assignment2/CarlTloz-CarlosSolar.ttl]: More prefixes should be added

File: Assignment2/CarlTloz-CarlosSolar.rdf [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+		<hasName>Pedro</hasName>
+    </rdf:Description>
+	
+    <rdf:Description rdf:about="#Computer101">
+		<hasOwner rdf:resource="#User10A"/>
+    </rdf:Description>
+	
+    <rdf:Description rdf:about="#Measurement8401">
+		<hasTemperature>29</hasTemperature>
+		<atTime>2010-06-12T12:00:12</atTime>
```
Comment (ocorcho) [inline Assignment2/CarlTloz-CarlosSolar.rdf]: Good, although it is good practice to datatype these literals

**Comment by CarlTloz** [comment]
Changes requested possibly resolved. Now Pedro is not a literal, I'm not really sure about how to treat Pedro, but it's not between quotation marks so I don't think it's a literal.

**Pull request closed without merging**